pkg/autoscaler/metrics/wire.proto

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Meta questions: why don't we follow the k8s model where proto is generated from the types (i.e. we already have `Stat` and it can be used to generate the proto file).

Reply:
See description. For example: The actual types contain `time.Time`. That's not even properly doable with "usual" go-protobuf. gogo-protobuf can do it, but I ran into a slew of other issues adding it to the types (i.e. then you depend on the google/protobuf types too, you need to pull them in etc etc.).

The simplest solution seemed to be to add the specific WireTypes which has the added benefit of dropping fields that we don't actually send over the wire (i.e. the timestamp).

Reply:
https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/time.go#L32

Reply:
Or as we talked, we can just dump time, since client side time is the only one that's used anyway.

Reply:
Done all that, we can now (almost) reuse the entire existing type hierarchy.